tests/functional/object/mcg/test_write_to_bucket.py

```diff
@@ -456,3 +459,42 @@ def test_content_encoding_with_write(
         logger.info(
             "Put object operation is preserving ContentEncoding as a object metadata"
         )
+
+    @bugzilla("2259189")
+    @bugzilla("2264480")
+    @pytest.mark.polarion_id("OCS-5773")
+    def test_nb_db_activity_logs_on_io(
+        self, bucket_factory, awscli_pod_session, mcg_obj
+    ):
+
+        """
+        This test checks if the activity logs are being logged
+        in the activitylogs table for every object upload and
+        deletion when noobaa log is set to default
+
+        """
+
+        bucket = bucket_factory()[0]
+        full_objectpath = f"s3://{bucket.name}"
+        logger.info("successfully created bucket")
+
+        sync_object_directory(
+            awscli_pod_session, AWSCLI_TEST_OBJ_DIR, full_objectpath, mcg_obj
+        )
+        logger.info("uploaded objects to the bucket")
```

Comment:
how many objects will it upload?

Reply:
actually im not sure. its everything that is present in the /test_objects directory